src/Parsers/ParserInsertQuery.cpp

```diff
@@ -174,6 +174,7 @@ bool ParserInsertQuery::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
         if (s_format.ignore(pos, expected) && !name_p.parse(pos, format, expected))
             return false;
     }
+    else if (infile) {}
```

Comment:
Cannot understand.

Reply:
Is because a syntax error is thrown when the query is executed, do you have any good suggestions.
```
INSERT INTO test_tb FROM INFILE /Users/didi/code/world.csv.bz2

Query id: 4add4203-7b9e-4cc1-9d39-39b4e45b3c36

Connecting to localhost:9000 as user default.
Connected to ClickHouse server version 21.13.1 revision 54455.


0 rows in set. Elapsed: 0.008 sec.

Received exception from server (version 21.13.1):
Code: 62. DB::Exception: Received from localhost:9000. DB::Exception: Syntax error: failed at position 71 (end of query): ;. Expected one of: COMPRESSION, FORMAT, SELECT, WITH, WATCH. (SYNTAX_ERROR)
```

Reply:
Let's  write a comment that it's needed to support syntax `INSERT INTO ... FROM INFILE ...` without `FORMAT` statement

Reply:
ok, i have added a comment.